src/metatensor/models/cli/eval_model.py

```diff
@@ -1,3 +1,46 @@
-def eval_model():
-    """evaluate a model"""
-    print("Run evaluate...")
+import argparse
+
+from ..utils.data.readers import read_structures
+from ..utils.data.writers import write_predictions
+from ..utils.model_io import load_model
+
+
+def _eval_model_cli(parser: argparse.ArgumentParser) -> None:
+    """Add the `eval_model` paramaters to an argparse (sub)-parser"""
+    parser.add_argument(
+        "-m",
```

Comment:
will this also provide a long form argument (i.e. `--model`)? If not I would like to have them, since they make bash scripts more self-documenting!

Reply:
No, it will not be there, but I can add this.

Reply:
I agree that we also need the long form of the arguments

Reply:
I added the options `--model` etc